css/src/tokens/layout.scss

```diff
@@ -4,9 +4,14 @@
 // Layout
 $column-gap: 0.75rem !default;
 $large-column-gap: 1.5rem !default;
-$reading-max-width: 50rem;
+$reading-max-width: 50rem !default;
 
-$layout-gap: 16px !default;
+$layout-gap: 24px !default; // 💀 needs new var
```

Comment:
Can't wait to see the VDs this change gonna cause 😅
Resolve after reading

Reply:
😈 It's time ;)